docs/assessment.md

```diff
@@ -260,13 +267,10 @@ PrairieLearn distinguishes between _assessments_ and _assessment instances_. An
 
 A student's percentage score will be determined by the number of points they have obtained, divided by the value of `maxPoints` for the assessment (subject to the rules associated to [`credit`](accessControl.md#credit) in assessment access rules).
 
-```json
-{
+```json title="infoAssessment.json"
```

Comment:
Why are we stripping the outer object here when we added it in the earlier JSON snippets?

Reply:
Not sure what happened here, looks like an oversight